packages/plugin-ext/src/hosted/browser/hosted-plugin.ts

```diff
@@ -404,7 +408,11 @@ export class HostedPluginSupport {
                 preferences: getPreferences(this.preferenceProviderProvider, this.workspaceService.tryGetRoots()),
                 globalState,
                 workspaceState,
-                env: { queryParams: getQueryParameters(), language: navigator.language },
+                env: {
+                    queryParams: getQueryParameters(),
+                    language: navigator.language,
+                    shell: await this.terminalService.getDefaultShell()
```

Comment:
could be mode to `Promise.all` above to fetch it in parallel

Reply:
Moved. Thank you for reviewing!